pkgs/README.md

```diff
@@ -67,7 +78,9 @@ To add a package to Nixpkgs:
 
    Some notes:
 
-   - All [`meta`](https://nixos.org/manual/nixpkgs/stable/#chap-meta) attributes are optional, but it’s still a good idea to provide at least the `description`, `homepage` and [`license`](https://nixos.org/manual/nixpkgs/stable/#sec-meta-license).
+   - Add yourself as the maintainer of the package.
+
+   - All other [`meta`](https://nixos.org/manual/nixpkgs/stable/#chap-meta) attributes are optional, but it’s still a good idea to provide at least the `description`, `homepage` and [`license`](https://nixos.org/manual/nixpkgs/stable/#sec-meta-license).
```

Comment:
platforms and mainProgram should be set, too.

Reply:
Personally I'm also expecting the license to be set, but let's have that conversation separately ok?

Reply:
Requiring license is probably defensible from «checking it is supposed to be packageable» point of view.

Reply:
This document is still a bit of a mess, there's also this section on meta which is probably a better place to change: https://github.com/NixOS/nixpkgs/blob/c5d067d0f23a3412312f036a315bd3e2a8970e9b/pkgs/README.md#meta-attributes

Reply:
I added the license question in the filters: d7b1941ca8c50ff500474bb864b634e921b41dd7

Agreed, once this is merged, I think we should re-format the doc with a focus on the contributor experience, and how they access that information.